<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="15" started-at="2016-02-15T07:39:48Z" finished-at="2016-02-15T07:39:48Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="15" started-at="2016-02-15T07:39:48Z" finished-at="2016-02-15T07:39:48Z">
      <class name="SecurityCheck.SecurityTest">
        <test-method status="FAIL" signature="US3674_XSSLocators(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:SecurityCheck.SecurityTest@41ee392b]" name="US3674_XSSLocators" duration-ms="0" started-at="2016-02-15T13:09:48Z" finished-at="2016-02-15T13:09:48Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Parameter 'TestScenerio' is required by @Test on method US3674_XSSLocators but has not been marked @Optional or defined
in C:\Users\dubve01\AppData\Local\Temp\testng-eclipse-1462636641\testng-customsuite.xml]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Parameter 'TestScenerio' is required by @Test on method US3674_XSSLocators but has not been marked @Optional or defined
in C:\Users\dubve01\AppData\Local\Temp\testng-eclipse-1462636641\testng-customsuite.xml
	at org.testng.internal.Parameters.createParameters(Parameters.java:155)
	at org.testng.internal.Parameters.createParameters(Parameters.java:358)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:450)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- US3674_XSSLocators -->
      </class> <!-- SecurityCheck.SecurityTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
